### DISCUSSION TITLE
Help(Again)

### DISCUSSION THREAD

**Discussion by Djsco42** [GitMyCode/Terraria-FishingBot#17] [Q&A]
So after learning a bit more about C# and wpf, I tried to preview MainWindow.xaml  and MainWindow.xaml.cs (as I was trying to start the app while in debug mode) and I came across this problem: "Some assembly references are missing. Building to restore the NuGet cache might resolve this issue." I tried to build the solution but it failed, I tried to get rid of as many error messages I got. Though I kept getting these Errors in Project: FishingBot.WindowsUI , file: GitVersionTask.targets and on lines 46 and 71 : DirectoryNotFoundException: Cannot find the .git directory

(Just ask if you want/need to see the full Error message.) I just don't know what to do here to fix this error. Any thoughts?

**Comment by GitMyCode** [comment]
Gitversion is the thing that automatically put a version with commits. You need to clone the repository I guess

**Reply by Djsco42** [reply]
Thanks a lot!
